torch/_export/__init__.py

```diff
@@ -169,9 +170,23 @@ def capture_pre_autograd_graph(
         # Do not decompose dropout for exported models, because in eval mode the dropout
         # op disappears from the graph, which makes it difficult to switch to train mode.
         # See https://github.com/pytorch/pytorch/pull/115258#issuecomment-1900755832.
+
+        # We force create native_batch_norm because the below materialization logic
+        # only applies to CIA ops.
+        maybe_aliasing_or_mutating_ops = [torch.ops.aten.native_batch_norm.default]
```

Comment:
it looks like you're hardcoding `aten.native_batch_norm` in two places: both here, and in your helper `_should_decompose_because_unsafe_op(op)`. Can't you just re-use that `_should_decompose_because_unsafe_op()` helper in ordre to generate this list?